dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ProcessDefinitionController.java

```diff
@@ -113,9 +126,10 @@ public Result createProcessDefinition(@ApiIgnore @RequestAttribute(value = Const
                                           @RequestParam(value = "tenantCode", required = true) String tenantCode,
                                           @RequestParam(value = "taskRelationJson", required = true) String taskRelationJson,
                                           @RequestParam(value = "taskDefinitionJson", required = true) String taskDefinitionJson,
+                                          @RequestParam(value = "otherParamsJson", required = false) String otherParamsJson,
```

Comment:
We should add this param to the swagger's doc.

Reply:
![1655366094749](https://user-images.githubusercontent.com/33045461/174021141-e2887f9a-9d2f-4b92-842b-fa37565f5962.png)

Reply:
This is not the swagger's param. You can take a look at ```@ApiImplicitParam```.